apps/webapp/src/components/staking/account/delegation-value-view/validator-info-component.tsx

```diff
@@ -0,0 +1,77 @@
+import { ValidatorInfo } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/stake/v1/stake_pb';
+import {
+  bech32IdentityKey,
+  calculateCommissionAsPercentage,
+  getIdentityKeyFromValidatorInfo,
+  getValidator,
+} from '@penumbra-zone/types';
+import {
+  Identicon,
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from '@penumbra-zone/ui';
+import { CopyToClipboardIconButton } from '@penumbra-zone/ui/components/ui/copy-to-clipboard-icon-button';
+
+export const ValidatorInfoComponent = ({
+  validatorInfo,
+  votingPowerAsIntegerPercentage,
+}: {
+  validatorInfo: ValidatorInfo;
+  votingPowerAsIntegerPercentage?: number;
+}) => {
+  const validator = getValidator(validatorInfo);
+  const identityKey = getIdentityKeyFromValidatorInfo(validatorInfo);
+
+  return (
+    <TooltipProvider delayDuration={0}>
+      <div className='flex items-center gap-4'>
+        <div className='shrink-0'>
+          <Identicon
+            uniqueIdentifier={bech32IdentityKey(identityKey)}
+            size={48}
+            className='rounded-full'
+            type='gradient'
+          />
```

Comment:
can this use the same identicon logic as the `ValueView` uses for rendering the delegation token in other places, so that the icon is the same regardless of whether it's rendered in the context of the staking page or in an asset list?

also, that way if we have a pre-registered Metadata for that delegation token (eg the validator operator sent us a PR) then we'll automatically use any logo they set.